[2025-06-01T17:12:47.583+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: My_dag.runme_0 manual__2025-06-01T17:12:40+00:00 [queued]>
[2025-06-01T17:12:47.591+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: My_dag.runme_0 manual__2025-06-01T17:12:40+00:00 [queued]>
[2025-06-01T17:12:47.591+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-06-01T17:12:47.611+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): runme_0> on 2025-06-01 17:12:40+00:00
[2025-06-01T17:12:47.624+0000] {standard_task_runner.py:57} INFO - Started process 567 to run task
[2025-06-01T17:12:47.634+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'My_dag', 'runme_0', 'manual__2025-06-01T17:12:40+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpme8sn4wn']
[2025-06-01T17:12:47.641+0000] {standard_task_runner.py:85} INFO - Job 31: Subtask runme_0
[2025-06-01T17:12:47.718+0000] {task_command.py:416} INFO - Running <TaskInstance: My_dag.runme_0 manual__2025-06-01T17:12:40+00:00 [running]> on host b77b72820179
[2025-06-01T17:12:47.829+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='My_dag' AIRFLOW_CTX_TASK_ID='runme_0' AIRFLOW_CTX_EXECUTION_DATE='2025-06-01T17:12:40+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-01T17:12:40+00:00'
[2025-06-01T17:12:47.830+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-01T17:12:47.832+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "My_dag__runme_0__20250601" && sleep 1']
[2025-06-01T17:12:47.849+0000] {subprocess.py:86} INFO - Output:
[2025-06-01T17:12:47.852+0000] {subprocess.py:93} INFO - My_dag__runme_0__20250601
[2025-06-01T17:12:48.858+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-06-01T17:12:48.898+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=My_dag, task_id=runme_0, execution_date=20250601T171240, start_date=20250601T171247, end_date=20250601T171248
[2025-06-01T17:12:48.934+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-01T17:12:48.959+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
